### PR TITLE
fix(Gmail Node): Add reply to email

### PR DESCRIPTION
**Pull request by feelgood-interface**
Changes:

- Add `replyTo` field for `draft` and `message`
- Split `additionalFields` for `send` and `reply` (because not all fields overlap)
- Fix `replayToEmail()` typo

### CODE REVIEW

**Comment by Joffcom** [comment]
`NODE-544`

File: packages/nodes-base/nodes/Google/Gmail/v1/DraftDescription.ts [hunk 1]
```diff
@@ -159,6 +159,14 @@ export const draftFields: INodeProperties[] = [
...
 				default: [],
 			},
+			{
+				displayName: 'Reply To Email',
+				name: 'replyTo',
+				type: 'string',
+				placeholder: 'reply@example.com',
+				default: '',
+				description: 'The email address that the reply message is sent to',
+			},
```
Comment (michael-radency) [inline packages/nodes-base/nodes/Google/Gmail/v1/DraftDescription.ts:169]: v1 exist only for compatibility, please do not modify it, unless it is a bug fix

File: packages/nodes-base/nodes/Google/Gmail/v2/MessageDescription.ts [hunk 2]
```diff
@@ -234,6 +234,83 @@ export const messageFields: INodeProperties[] = [
...
+				default: '',
+				description: "The name that will be shown in recipients' inboxes",
+			},
+			{
+				displayName: 'Reply To Email',
+				name: 'replyTo',
+				type: 'string',
+				placeholder: 'reply@example.com',
+				default: '',
+				description: 'The email address that the reply message is sent to',
```
Comment (michael-radency) [inline packages/nodes-base/nodes/Google/Gmail/v2/MessageDescription.ts:310]: no need to duplicate all options, you can display option conditionally to operation
```
		displayOptions: {
			show: {
				'/operation': ['send'],
			},
		},
		displayOptions: {
			hide: {
				'/operation': ['reply'],
			},
		},
```

**Comment by michael-radency** [comment]
Hello @feelgood-interface , thank you for your contribution, I did add some comments

**Comment by michael-radency** [comment]
Hello @feelgood-interface , I did update branch with master, update some wording, removed `displayOptions` from `replyToSenderOnly` to avoid breaking change. If everything good please let me know so we could merge this PR

**Review by michael-radency — APPROVED** [review]

**Comment by michael-radency** [comment]
Hello @feelgood-interface , I could not merge this PR due to issues with tests, I did open new https://github.com/n8n-io/n8n/pull/6453 on n8n-io

**Pull request closed without merging**